.travis.yml

```diff
@@ -35,7 +35,7 @@ env:
     - SYMFONY_DEPRECATIONS_HELPER=weak_vendors
     - TARGET=test
     - UPSTREAM_URL=https://github.com/sonata-project/exporter.git
-    - PHPUNIT_VERSION=7
+    - PHPUNIT_VERSION=8
```

Comment:
isn't this managed by dev-kit?

Reply:
just read the description, ignore me

Reply:
By "tonight" I mean that it should change the version in another repo tonight. If that works, we shall make other PRs to migrate more repos to phpunit 8